bitnami/flink/templates/jobmanager/deployment.yaml

```diff
@@ -0,0 +1,160 @@
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.jobmanager.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+      {{- if .Values.jobmanager.initContainers }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ include "flink.jobmanager.fullname" . }}
+          image: {{ include "flink.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.jobmanager.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.jobmanager.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
+          {{- else if .Values.jobmanager.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.jobmanager.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.args "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.jobmanager.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.jobmanager.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: tcp-rpc
+              containerPort: {{ .Values.jobmanager.containerPorts.rpc }}
+            - name: http
+              containerPort: {{ .Values.jobmanager.containerPorts.http }}
+            - name: tcp-blob
+              containerPort: {{ .Values.jobmanager.containerPorts.blob }}
+          env:
+            - name: FLINK_MODE
+              value: jobmanager
+            - name: FLINK_CFG_JOBMANAGER_RPC_ADDRESS
+              value: {{ include "flink.jobmanager.fullname" . | quote }}
+            - name: FLINK_CFG_JOBMANAGER_RPC_PORT
+              value: {{ .Values.jobmanager.service.ports.rpc | quote }}
+            - name: FLINK_CFG_JOBMANAGER_RPC_BIND__PORT
+              value: {{ .Values.jobmanager.containerPorts.rpc | quote }}
+            - name: FLINK_CFG_JOBMANAGER_BIND__HOST
+              value: 0.0.0.0
+            - name: FLINK_CFG_REST_ADDRESS
+              value: {{ include "flink.jobmanager.fullname" . }}
```

Comment:
Quote it. If someone uses fullnameOverride and it only contains numbers the deployment will fail

Reply:
[09b240e](https://github.com/bitnami/charts/pull/14879/commits/09b240e396d2a547b6e40db1c98d3f90c77f0814)